pkg/controller/kubernetes/containerregistry.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2021 The Crossplane Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes
+
+import (
+	"context"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/crossplane/crossplane-runtime/pkg/meta"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/digitalocean/godo"
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/crossplane-contrib/provider-digitalocean/apis/kubernetes/v1alpha1"
+	do "github.com/crossplane-contrib/provider-digitalocean/pkg/clients"
+	dok8s "github.com/crossplane-contrib/provider-digitalocean/pkg/clients/kubernetes"
+)
+
+const (
+	// Error strings.
+	errNotContianerRegistiry             = "managed resource is not a DOContainerRegistery resource"
```

Comment:
Can you go through this file and fix the spelling of Registry, I see both `Registiry` and `Registery`.

Reply:
sure, sorry for missing these